### ISSUE TITLE
Iso

### ISSUE THREAD

**Issue by mrmajik45** [Meulengracht/MollenOS#176]
Can we build Iso's yet? 

- [ ] Build

**Comment by Meulengracht** [comment]
Yes it is supported, however discutility does not expose the option to do so. You can build .img and .vmdk files through "make install_vmdk" and "make install_img"

**Comment by mrmajik45** [comment]
No I mean for the OS, Can the OS run yet? If so can we have a "release" as github says

**Comment by Meulengracht** [comment]
Yes the OS runs, i have an official release planned very soon for MollenOS. There are still some bugs for stability and overall feel, but if you cant wait you can clone and build it for yourself :-)

**Comment by Meulengracht** [comment]
Yes i do, its a Windows clone, but its an awesome project

**Comment by mrmajik45** [comment]
Agreed. Happy they got BTRFS support.
